viral/grosmark_analysis.py

```
@@ -240,37 +290,75 @@ def offline_correlations(
-    )
+        correlations_vs_peak_distance(
+            real_corrs, peak_position_cm=peak_position_cm, plot=True
+        )
+    else:
+        offline_spks_pre, offline_spks_post = get_preactivation_reactivation(
+            flu=spks, wheel_freeze=wheel_freeze
+        )
+        pre_corrs_real = get_offline_correlation_matrix(
+            offline=offline_spks_pre, wheel_freeze=True, do_shuffle=False, plot=True
+        )
+        pre_corrs_shuffled = get_offline_correlation_matrix(
```

Ruff (0.8.2)

Local variable `pre_corrs_shuffled` is assigned to but never used

Remove assignment to unused variable `pre_corrs_shuffled`

(F841)